app/services/ConsignmentService.scala

```diff
@@ -53,4 +55,11 @@ class ConsignmentService @Inject()(val graphqlConfiguration: GraphQLConfiguratio
     sendApiRequest(getConsignmentFolderDetailsClient, getConsignmentFolderDetails.document, token, variables)
       .map(data => data.getConsignment.get)
   }
+
+  def getConsignmentTransferSummary(consignmentId: UUID, token: BearerAccessToken): Future[getConsignmentSummary.GetConsignment] = {
+    val variables: getConsignmentSummary.Variables = new getConsignmentSummary.Variables(consignmentId)
+
+    sendApiRequest(getConsignmentSummaryClient, getConsignmentSummary.document, token, variables)
+      .map(_.getConsignment.get)
```

Comment:
Does the `_.getConsignment.get` function similarly to `data => data.getConsignment.get` in the above methods? Just wondering if theres a difference in functionality with that syntax?

Reply:
Ah, yes, you're right!, I'll make the "data" one look like the underscore one. Thanks